src/io/json.jl

```diff
@@ -50,52 +54,118 @@ function _parse_mats!(root_dict)
 end
 
 
-""
-function parse_json(file::String; kwargs...)
-    pmd_data = open(file) do io
-        parse_json(io; filetype=split(lowercase(file), '.')[end], kwargs...)
+"parses enums from json"
+function _parse_enums!(data::Dict{String,<:Any})
```

Comment:
For now this is a good solution. Let's revisit whether we can still save and load data models after the release with the features in v10 without breaking them